### PR TITLE
Configure workload access outside cluster

### PR DESCRIPTION
**Pull request by bmckercher123**
New How to

- Final review from Matt D
- Can merge at any time

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit cab427d3f3531302cee6cb0d0c0ce83d4c49b8f5

https://deploy-preview-2908--calico.netlify.com

File: master/networking/workloads-outside-cluster.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **IPPool** resource with natOutgoing field
+
+### Concepts
+
+#### Outgoing NAT on each node
+
+For all networking options, you can allow your workloads to access the internet (subject to network policy, of course!) by enabling outbound NAT on the IP pool.  When enabled, traffic from pods in that pool go to a destination outside all {{site.prodname}} pools will be NATed.  The source address is masqueraded to an IP of the node on which each workload is hosted, allowing the network to understand the traffic. By default, the NAT outgoing option is automatically enabled for the pool that is created when you install {{site.prodname}}.
```
Comment (lmm) [inline master/networking/workloads-outside-cluster.md]:
```suggestion
For all networking options, you can allow your workloads to access the internet (subject to network policy, of course!) by enabling outbound NAT on the IP pool.  When enabled, traffic from pods in that pool going to a destination outside all {{site.prodname}} pools will be NATed.  The source address is masqueraded to an IP of the node on which each workload is hosted, allowing the network to understand the traffic. By default, the NAT outgoing option is automatically enabled for the pool that is created when you install {{site.prodname}}.  
```

Comment (lmm) [reply master/networking/workloads-outside-cluster.md]: May be hard to see but the change above is:
```traffic from pods in that pool go to a destination outside all {{site.prodname}} pools will be NATed```
to
```traffic from pods in that pool going to a destination outside all {{site.prodname}} pools will be NATed```

File: master/networking/workloads-outside-cluster.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+#### BGP peering with out-of-cluster NAT
+
+When {{site.prodname}} is [BGP peered with your physical network infrastructure]({{site.baseurl}}/{{page.version}}/networking/bgp), you can use the infrastructure to NAT traffic from pods to the internet.  In this case, you should disable the {{site.prodname}} NAT outgoing option. 
+
+#### Expose pods directly on the internet
+
+If you want your pods to have public internet IPs:
+
+- Configure {{site.prodname}}to peer with your physical network infrastructure
```
Comment (lmm) [inline master/networking/workloads-outside-cluster.md]:
```suggestion
- Configure {{site.prodname}} to peer with your physical network infrastructure
```

File: master/networking/workloads-outside-cluster.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- Create an IP pool for those pods that contain public IP addresses that are routed to your network with `nat-outgoing: false`
```
Comment (lmm) [inline master/networking/workloads-outside-cluster.md]:
```suggestion
- Create an IP pool for those pods that contain public IP addresses that are routed to your network with `natOutgoing: false`
```

**Pull request merged**